types/hooks/screens/useChatScreenHook.types.ts

```diff
@@ -13,6 +13,7 @@ interface UseChatScreenHook {
   selfAnonUserInfo: UserSchema | null;
   handleUserName: (item: ChatSchema) => string;
   updateChatContinuity: (chatsData: ChatSchema[]) => ChatSchema[];
+  setIsLoadingFetchAllMessage: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `setIsLoadingFetchAllMessage` function in the `UseChatScreenHook` interface is a good approach to manage the loading state during the fetching of all messages. However, it's important to ensure that this function is used properly throughout the codebase to avoid any inconsistencies or unexpected behavior. Also, make sure to handle potential errors when setting the state.
```diff
+  setIsLoadingFetchAllMessage: React.Dispatch<React.SetStateAction<boolean>>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->